midpoint.csv

```diff
@@ -0,0 +1,3 @@
+,a_new,b
```

Comment:
Curious to hear, what is the purpose of this file? If it is not necessary, how did it arise, and can we delete it?

Reply:
hmmm, no idea what i was doing to get this :grin:, will delete